frontend/src/utils/getBannerProps.ts

```diff
@@ -0,0 +1,32 @@
+import { BannerVariant } from '~theme/components/Banner'
+
+export type BannerProps = {
+  variant: BannerVariant
+  msg: string
+}
+
+export const getBannerProps = (bannerContent?: string): BannerProps | null => {
+  if (!bannerContent) return null
+  if (bannerContent.startsWith('info:')) {
+    return {
+      variant: 'info',
+      msg: bannerContent.slice(5),
```

Comment:
should we trim the content after slicing, so the banner will not have leading/ending spaces

Reply:
yes! thanks for catching it. added in [369db25](https://github.com/opengovsg/FormSG/pull/4610/commits/369db25ac2586413b5d778e73e185a08acced32a)